### PR TITLE
fix: use new editor in more places (DE44657)

### PR DESCRIPTION
**Pull request by devpow112**
We missed a few spots when pluming through `use-new-brightspace-editor`. This was causing the old editor to get loaded as well as the new one and v4 and v5 do not play well together. It's mainly death, lots of death, if they are both loaded on the same page. 
[White-space diff off](https://github.com/Brightspace/d2l-rubric/pull/966/files?w=1).

Corresponding FACE change: https://github.com/BrightspaceHypermediaComponents/activities/pull/1934.

### CODE REVIEW

File: editor/d2l-rubric-html-editor.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ Polymer({
 			style="[[_getEditorStyle(_editorActive)]]"
 			type="[[_getType(editorType)]]"
 			html$={{value}}
-			no-border
+			no-border$={{!border}}
```
Comment (devpow112) [inline editor/d2l-rubric-html-editor.js:63]: This was needed cause we were disabling the border for all HTML boxes but the one below the rubrics in the options section needed it's border.

File: editor/d2l-rubric-editor.js [hunk 2]
```diff
@@ -377,8 +377,10 @@ const $_documentContainer = html `
 								pending-saves="[[_pendingDescriptionSaves]]"
 								on-text-changed="_saveDescription"
 								rich-text-enabled="[[_richTextAndEditEnabled(richTextEnabled,_canEditDescription)]]"
+								use-new-brightspace-editor="[[useNewBrightspaceEditor]]"
 								html-editor-type="full"
-								html-editor-height="12rem">
+								html-editor-height="12rem"
+								border>
```
Comment (devpow112) [inline editor/d2l-rubric-editor.js:383]: This is where it needed a boarder

File: editor/d2l-rubric-html-editor.js (hunk 1 above)
Comment (semibran) [reply editor/d2l-rubric-html-editor.js:63]: Nothing like a little styles in HTML :relaxed:

File: editor/d2l-rubric-html-editor.js [hunk 3]
```diff
@@ -48,8 +48,8 @@ Polymer({
 				margin: 2px;
 			}
 
-			d2l-htmleditor:focus-within,
-			d2l-htmleditor:hover {
+			d2l-htmleditor[no-border]:focus-within,
+			d2l-htmleditor[no-border]:hover {
```
Comment (semibran) [inline editor/d2l-rubric-html-editor.js:52]: Was trying to figure out why there was a border definition under a selector with no-border. But I think I figured it out lol.

**Review by semibran — APPROVED** [review]

File: editor/d2l-rubric-html-editor.js (hunk 3 above)
Comment (devpow112) [reply editor/d2l-rubric-html-editor.js:52]: Yeah it's a bit confusing but when the HTML editor has no boarder we add our own custom border.

**Pull request merged**